geoprocessing/src/modules/shapefiles/shapefiles.service.ts

```diff
@@ -0,0 +1,70 @@
+import { Injectable, Logger } from '@nestjs/common';
+
+import { createReadStream, rmdirSync, unlinkSync } from 'fs';
+import { Extract } from 'unzipper';
+const mapshaper = require('mapshaper');
+
+@Injectable()
+export class ShapeFileService {
+  private readonly logger: Logger = new Logger(ShapeFileService.name);
+  constructor() {}
+
+  private unzipShapefile(fileInfo: Express.Multer.File): Promise<string> {
+    return new Promise((resolve, reject) => {
+      createReadStream(fileInfo.path)
+        .pipe(
+          Extract({
+            path: `${fileInfo.destination}/${fileInfo.filename.replace(
+              '.zip',
+              '',
+            )}`,
+          }),
+        )
+        .on('close', () =>
+          resolve(`${fileInfo.filename} extracted succesfully`),
+        )
+        .on('error', (error: Error) =>
+          reject(
+            new Error(
+              `${fileInfo.filename} could not been extracted: ` + error,
+            ),
+          ),
+        );
+    });
+  }
+
+  private async shapeFileToGeoJson(fileInfo: Express.Multer.File) {
+    const outputKey = `shapefile-${new Date().getTime()}.geojson`;
+
+    const _geoJson = await mapshaper.applyCommands(
+      `-i ${fileInfo.path.replace(
+        '.zip',
+        '',
+      )}/*.shp -info -o ${outputKey} -verbose `,
```

Comment:
I believe it should support `mute_all_logs` setting.

Reply:
Deleted verbose flag [here](https://github.com/Vizzuality/marxan-cloud/pull/161/commits/8dee9241dc41373ccd309bbd3d104b72f93e10ea)